packages/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -225,7 +225,8 @@ class Transforms {
           return Promise.resolve(object);
         }
 
-        if (!container.onBehalfOfUser) {
```

Comment:
I've let the flow continue for encryptedTextKeyValues when there is on onBehalfOf present. This is the correct use case for CDR compliance which uses org wide keys.